login.html

```diff
@@ -1 +1,58 @@
-login.html
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/reset-css@4.0.1/reset.min.css"
+    />
+    <link
+    rel="stylesheet" href="./styles/login/">
+    <link rel="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+</head>
+<body>
+
+    <!-- 메인 컨텐츠 -->
+    <main class="main">
+        <header>
+            <a href="/">
+                <img src="images/logo.png" alt="판다 마켓" class="container__logo" />
+            </a>
+        </header>
+    <!-- 아이디 비밀번호 박스 -->
+        <div class="container">
+            <div class="email-box">
+                <label class="email-box__text">이메일</label>
+                <input class="email" placeholder="   이메일을 입력해주세요" type="email" onfocus="this.placeholder = ''" />
```

Comment:
```suggestion
                <input class="email" placeholder="이메일을 입력해주세요" type="email" />
```
1. placeholder를 옆으로 두고싶으셔서 띄어쓰기를 추가하신것 같아요! 그렇게 되면 placeholder는 옆에 있지만 실제 값을 넣으면 간격이 placeholder와 다르게 들어가서 통일 되어 보이지 않을 수 도 있습니다. 한번 css로 해결을 해보시는 것은 어떠실까요? 
2. onfocus가 되면 placeholder가 사라지게 잘 해주신 것 같아요! 그런데 placeholder는 입력값이 생긴다면 자동으로 없어지고, 아무것도 없는상태에서 focus가 disable되면 다시 나타나는 기본 기능을 가지고 있어요! 현재는 focusout이 되면 placeholder는 다시 생기지 않는것 같아요. 혹시 focus가 되었을 때 placeholder를 사라지게 하고싶으시다면 추후 focus가 out되었을 때, 다시 생길 수 있도록 추가해주시는 것이 사용자 경험상 좋을 것 같아요 :)